### PR TITLE
fix(office): fixes user invitation to external meeting opening jitsi instead

### LINKED ISSUE #222: When click "invite user to a meet" and have a externalMeetURL, matrix open "JitsiMeet"
## Description
I use matrix in my company and all rooms have external meet url (meet.google.com). When we click in "Invite user to a meet", user confirm and enter in a Jitsi Meet.

Always when a user accepct invitation have a console log that show externalMeetURL variable value. App is logging correctly the external meet url but open a jitsi meet

### Steps to Reproduce
1. Enter a room that have externalMeetURL
2. Invite a user

### Expected Behavior
- Open external meet link

### Current Behavior
- Open Jitsi Meet

### PR DESCRIPTION
**Pull request by phiter**
### Description

This fixes a bug where when inviting users to join an external meeting, a jitsi room was created instead.

closes #222
### How to test?

1. Open matrix in two different accounts
2. Join an external meeting in one account
3. Invite the other user to join the same meeting
4. Accept the invitation as the other user
### Expected behavior

The user should join the external meeting and no jitsi call should be created for that room.

### CODE REVIEW

**Comment by phiter** [comment]
I decided to take the shortest path to fix this one since it is something my company needs to use. I extracted the `onEnterRoom` function and exported it, then imported it to be used where the accept invitation event happens.

I think that function should be extracted to another file instead of `OfficePage.js` but I don't know where exactly, so please review it and let me know where is the best place to put it.

**Comment by angeliski** [comment]
Nice @phiter 
Maybe can we move that functions to the store?
What do you think @megatroom? React isn't my strongest point

File: frontend/src/morpheus/containers/OfficePage.js [hunk 1]
```diff
@@ -54,33 +84,9 @@ const OfficePage = ({
...
-
-              }else{
-                const redirectUrl = `/morpheus/room/${room.id}`;
-                if (event.ctrlKey) {
-                  window.open(redirectUrl, "_blank");
-                } else {
-                  history.push(redirectUrl);
-                }
-              }
+              enterRoom(room, event, event.ctrlKey);
```
Comment (megatroom) [inline frontend/src/morpheus/containers/OfficePage.js:89]: You call with params `room, event, event.ctrlKey` and the method receives `room, history, openInNewTab`.

- `event.ctrlKey` is a boolean?
- `event` isn't a history.

Comment (phiter) [reply frontend/src/morpheus/containers/OfficePage.js]: Hey! Sorry about event, it should be `history` instead. I've fixed it, also extracted `event.ctrlKey` into a variable to better represent its meaning.

Yes, `event.ctrlKey` is a boolean.

**Comment by phiter** [comment]
@juliemar done!

**Pull request merged**